src/Foundation/NSUrlSessionHandler.cs

```diff
@@ -317,6 +319,17 @@ protected override void Dispose (bool disposing)
 			}
 		}
 
+		NSUrlSessionHandlerTrustOverrideForUrlCallback trustOverrideForUrl;
```

Comment:
I don't love the ForUrl suffix, it looks obj-c ish, but I have no better suggestion.